gossip/api/types_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"testing"
+)
+
+func TestTimestampedEntryHash(t *testing.T) {
+	tests := []struct {
```

Comment:
nit: 

The tests can be inlined with the for loop:

```go 
for _, test := range []struct{
    dec string
     ...
}{
     { /* test case 1 */ },
}{
    /* test body */
}
```

Reply:
That seems like a personal preference, and personally I prefer the current layout :-)